### PR TITLE
Fixed real world example

### PR DESCRIPTION
**Pull request by garethbradley**
There were a couple of issues with the real-world example.

Use of lodash/array and lodash/object has changed and results in module not found error

Change to Github API? Keys are id numbers, not login names. Updated UserPage and RepoPage to find  IDs

### CODE REVIEW

File: examples/real-world/yarn.lock [hunk 1]
```diff
@@ -0,0 +1,3805 @@
+# THIS IS AN AUTOGENERATED FILE. DO NOT EDIT THIS FILE DIRECTLY.
```
Comment (Andarist) [inline examples/real-world/yarn.lock]: this file should be removed, while you are at it you could add `yarn.lock` to `.gitignore`d files

File: examples/real-world/containers/RepoPage.js [hunk 2]
```diff
@@ -6,6 +6,7 @@ import Repo from '../components/Repo'
 import User from '../components/User'
 import List from '../components/List'
 import PropTypes from 'prop-types'
+import object from 'lodash/object'
```
Comment (Andarist) [inline examples/real-world/containers/RepoPage.js]: could we just switch to using `lodash` entry directly in those places? like `import { findKey } from 'lodash'`

Comment (garethbradley) [reply examples/real-world/containers/RepoPage.js]: I'm no expert - I'd have to try it. I'm guessing the syntax would look like:

`import { findKey } from 'lodash/object';`

Comment (Andarist) [reply examples/real-world/containers/RepoPage.js]: actually `lodash` exports everything from the "main entry" too, u dont have to "scope" it under `object/lodash`

if you dont feel confident in doing this, let me know and I'll merge this as is

Comment (garethbradley) [reply examples/real-world/containers/RepoPage.js]: There we go - that should be sorted.

Cheers

**Comment by Andarist** [comment]
Thanks!

**Pull request merged**